### ISSUE TITLE
`data-pdfmake` with table

### ISSUE THREAD

**Issue by Aymkdn** [Aymkdn/html-to-pdfmake#56]
I can't repeat this example:

```
<div data-pdfmake="{&quot;layout&quot;:&quot;noBorders&quot;}">
    <table><tr><td>Hello</td><td>World</td></tr></table>
 </div>
```

In the log I get the following code:

```
[
  { text: '﻿', style: [] },
  { nodeName: 'DIV', stack: [ [Object] ], layout: 'noBorders' }
]
```

_Originally posted by @etoktaev in https://github.com/Aymkdn/html-to-pdfmake/issues/26#issuecomment-646465396_

**Comment by Aymkdn** [comment]
Hi,

I changed a bit how it works: I've just published v2.0.6 and you now have to write it this way:
```html
<div>
    <table data-pdfmake="{&quot;layout&quot;:&quot;noBorders&quot;}"><tr><td>Hello</td><td>World</td></tr></table>
 </div>
```

**Comment by etoktaev** [comment]
It works! Thank you very much!
